README.md

```diff
@@ -0,0 +1,57 @@
+# Lambda Function for ingesting syslog-over-https
+
+This lambda function ingests syslog-over-https with basic auth, specifically targetting Heroku
+logdrains.
+
+It uses a DynamoDB table to store username/password combinations, with the password being
+bcrypt hashed. After validating that a request has a working use of credentials, it will
+send the logs to a Kinesis stream, one log line per record.
```

Comment:
What are the username/passwd combos for? Any reason we're using DynamoDB in particular?

Reply:
>What are the username/passwd combos for?

syslog-over-https within Heroku (and I believe other systems as well) only supports Basic Auth, so we are using that for authentication for log ingestion.

>Any reason we're using DynamoDB in particular?

Just because it's simple, but per https://github.com/mozilla-services/pylogdrain/pull/1#discussion_r208775772 I'm going to refactor it to use a JSON doc in S3 instead.